Makefile

```diff
@@ -139,7 +158,7 @@ fluentd:
 image-cleanup-test-images:
 	cd misc/image-cleanup-test-images; $(MAKE) $(MFLAGS)
 
-get-deps:
+get-deps: get-cni-sources
```

Comment:
Even though, the cni-plugin package is not required for development, as it should be developed outside agent package. The cni-plugin package is only required for building where the dependency has been added to `docker` and `docker-release`.